hail/python/hail/table.py

```diff
@@ -3750,7 +3750,7 @@ def summarize(self, handler=None):
 
     @typecheck_method(parts=sequenceof(int), keep=bool)
     def _filter_partitions(self, parts, keep=True) -> 'Table':
-        return Table(ir.TableToTableApply(self._tir, {'name': 'TableFilterPartitions', 'parts': parts, 'keep': keep})).persist()
+        return Table(ir.TableToTableApply(self._tir, {'name': 'TableFilterPartitions', 'parts': parts, 'keep': keep}))
```

Comment:
this seems crazy?

Reply:
the persist was crazy, yes. No persist, no crazy.